lib/facility.rb

```diff
@@ -1,14 +1,65 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_accessor :name, 
+              :address, 
+              :phone, 
+              :services,
+              :registered_vehicles, 
+              :collected_fees
```

Comment:
We should only be using `attr_accessor` in instances where we intend for an instance variable to be read AND overwritten from outside of the class. None of the instance variables meet this criterion and only need to be read from outside the class, therefore we should be using an `attr_reader` instead.